Content.Shared/Movement/Pulling/Systems/PullingSystem.cs

```diff
@@ -152,38 +152,28 @@ private void OnAfterState(Entity<PullerComponent> ent, ref AfterAutoHandleStateE
 
     private void OnDropHandItems(EntityUid uid, PullerComponent pullerComp, DropHandItemsEvent args)
     {
-        if (pullerComp.Pulling == null || pullerComp.NeedsHands)
+        if (pullerComp.NeedsHands)
             return;
 
-        if (!TryComp(pullerComp.Pulling, out PullableComponent? pullableComp))
-            return;
-
-        TryStopPull(pullerComp.Pulling.Value, pullableComp, uid);
```

Comment:
TryStopPull will get his PullableComponent no need to it verbose.
pullerComp.Pulling == null is also done inside TryStopPull